src/test/java/uk/gov/hmcts/reform/migration/service/RespondentsDataMigrationServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.migration.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.domain.OldRespondent;
+import uk.gov.hmcts.reform.domain.OldRespondents;
+import uk.gov.hmcts.reform.domain.common.Address;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RespondentsDataMigrationServiceTest {
+
+    private RespondentsDataMigrationService service = new RespondentsDataMigrationService();
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void shouldMapCaseDetailsToNewCaseDetails() {
```

Comment:
Having one test for migration seems way too little to give us confidence that data transformation works and is bullet proof. I would not relay solely on manual testing when we handle live data.